src/blocks/Contributors/style.scss

```
@@ -0,0 +1,172 @@
+
+				.default-icon {
+					display: flex;
+					align-items: center;
+					justify-content: center;
+					background-color: #0073aa;
+					color: white;
+
+					svg {
+						width: 60%;
+						height: 60%;
+						fill: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)